### ISSUE TITLE
NTR: Time owned personal bednet categorization

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB-ontology/VEuPathDB-ontology#446]
For new term requests, please provide the following information:
**Term Label** [required]
Time owned personal bednet categorization

**Synonyms** [optional]

**Definition** [required except for organism name]
Categorization of values from EUPATH_0024266 - Time owned personal bednet

**Suggested Parent Term** [required]
Please look in the ontology hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/eupath) or choose one from following list: 
- [ ] Material (independent of other entities and can be maintained through time, such as person, cell, sample)
- [ ] Process (has temporal proper parts and for some time depends on some material entity, such as life cycle, cell division, heart beating, sleeping)
- [ ] Information (generically dependent on some artifact and stands in relation of aboutness to some entity, such as data, identifier, document)
- [ ] Characteristic/Quality (inheres in or is borne by other entities, such as height, color, disease)

**Examples** [optional]

**Contributor** [optional]
If you would like to be a contributor of the term or anyone you think that contributes to the term, such as label, definition, please provide the name and ORCID ID.

**Requester** [optional]
Sheena/ClinEpiDB

**Comment by johnwjudkins** [comment]
@SheenaTomko 

EUPATH:0024266
label: time of bed net ownership
definition: A time measurement datum of the duration a human being has owned a bed net.
parent: time measurement datum

EUPATH:0025175
label: categorical time of bed net ownership
definition: A categorical measurement datum that is the specified output of a data transformation having time of bed net ownership as input.
parent: categorical measurement datum

**Comment by SheenaTomko** [comment]
Looks good to me @johnwjudkins!

**Comment by zhengj2007** [comment]
Based on the definition, shall we label the term as 'duration of bed net ownership'?

**Comment by zhengj2007** [comment]
EUPATH:0025175
label: categorical time of bed net ownership

The term is added with label 'categorical duration of bed net ownership'. Is it the label we'd like use? @johnwjudkins

**Comment by johnwjudkins** [comment]
> EUPATH:0025175 label: categorical time of bed net ownership
> 
> The term is added with label 'categorical duration of bed net ownership'. Is it the label we'd like use? @johnwjudkins

Yes, because that term references 'duration of bed net ownership' (EUPATH:0024266).
